src/gui/model.rs

```diff
@@ -449,6 +450,22 @@ impl Greeter {
         for (k, v) in env {
             environment.push(format!("{}={}", k, v));
         }
+
+        self.updates.set_message("Logging in...".to_string());
```

Comment:
We can remove this message, since it'll never be shown. That's because if it succeeds, then we immediately exist. If it fails, we reset the message and then show the error message. In all cases, we never exit the function without the "Logging in..." message, so Relm4 never picks it up.